### ISSUE TITLE
Plot.pointer should compose with a mark’s render transform

### ISSUE THREAD

**Issue by Fil** [observablehq/plot#1614]
Plot.pointer [overwrites an existing render transform](https://github.com/observablehq/plot/blob/3e9497ad72157f8c54bd20c25580abb6961e70b2/src/interactions/pointer.js#LL18C1-L19C1). Seems to me that, instead, it should compose with it.

To reproduce, go to https://observablehq.com/@observablehq/plot-pole-of-inaccessibility-1587@200 and check the show ellipses checkbox. The dot mark’s render (which makes ellipses instead of dots) is then ignored.

Not sure if this qualifies as a bug (or latent bug), but probably easy to fix. I'll try tomorrow.

**Comment by mbostock** [comment]
Agree, it’s a bug. But also we haven’t documented render transforms yet so I don’t expect to encounter this one much!
